maintenance/datastore-migration.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Migrate Calico data from an etcdv3 datastore to a Kubernetes datastore
+description: Migrate your cluster from using an etcdv3 datastore to a Kubernetes datastore.
+canonical_url: '/maintenance/datastore-migration'
+---
+
+## Big picture
+
+Switch your {{site.prodname}} datastore from etcdv3 to Kubernetes.
+
+## Value
+
+To utilize all of {{site.prodname}}'s newest features, Kubernetes datastore
```

Comment:
> To utilize all of {{site.prodname}}'s newest features, Kubernetes datastore
is required.

Not entirely true. 

I think this should say something like this:

"Using Kubernetes as your datastore provides a number of benefits over using etcdv3 directly, including fewer components and better support for role based access control. For most users, using the Kubernetes data store will provide a better experience."